src/data/multiset/basic.lean

```diff
@@ -37,6 +37,7 @@ instance has_decidable_eq [decidable_eq α] : decidable_eq (multiset α)
 | s₁ s₂ := quotient.rec_on_subsingleton₂ s₁ s₂ $ λ l₁ l₂,
   decidable_of_iff' _ quotient.eq
 
+/-- defines a size for a multiset by referring to the size of the underlying list -/
 protected def sizeof [has_sizeof α] (s : multiset α) : ℕ :=
```

Comment:
This is better. I suggest you refer to the `multiset` module for examples. The definitions are done using a term and the proof only uses tactics.

Reply:
I made the definition in term mode and extracted the property that two permutations have the same size into a separate theorem.